website/docs/gs/calling_modes.md

```diff
@@ -20,7 +20,8 @@ use cases:
 
 Users should also consider that the single-sample mode is provided as a single workflow and is therefore considerably 
 simpler to run than joint calling. However, it also has higher compute costs on a per-sample basis and will not be as sensitive 
-as joint calling with larger cohorts.
+as joint calling with larger cohorts. Additionally, SV quality will be best when the case sample closely resembles the samples
+in the reference panel.
```

Comment:
```suggestion
as joint calling with larger cohorts. Additionally, input sequencing data should of similar quality to that of the reference panel, which was generated in [Byrska-Bishop et al. 2022](https://doi.org/10.1016/j.cell.2022.08.004) and is hosted in [this AnVIL workspace](https://app.terra.bio/#workspaces/anvil-datastorage/1000G-high-coverage-2019).
```

Reply:
The reason I worded this without referring to our provided reference panel is because we do also give information on how users can build their own reference panel. I recognize most won't, but I felt like it was sufficient to state that samples should resemble the reference panel, and the page describes the provided sample above

Reply:
Ok if this is redundant don't worry about it, but I think the word "resemble" is a bit vague so just make it clear you're referring to sample quality and sequencing methods.